lib/content-services/search/components/search-number-range/search-number-range.component.spec.ts

```diff
@@ -124,4 +125,11 @@ describe('SearchNumberRangeComponent', () => {
         expect(context.queryFragments['range1']).toEqual('cm:content.size:<0 TO 100>');
     });
 
+    it('should throw "pattern" error if FROM value is formed by letters', () => {
+        component.from = new FormControl(123);
+
+        component.ngOnInit();
+
+        expect(component.from.hasError('pattern'));
```

Comment:
What are we trying to check? Expect what? to be true to be false?
I was expecting to have a unit test like 

component.from = new FormControl('abc');
expect(component.from.hasError('pattern').toBe(true));


component.from = new FormControl(123);
expect(component.from.hasError('pattern').toBe(false));

component.from = new FormControl();
expect(component.from.hasError('required').toBe(true));

component.from = new FormControl(123);
expect(component.from.hasError('required').toBe(false));